neo4j/internal/metadata/metadata.go

```diff
@@ -17,4 +17,4 @@
 
 package metadata
 
-const DriverVersion = "5.28.1"
+const DriverVersion = "6.0.0-alpha"
```

Comment:
Maybe
```suggestion
const DriverVersion = "6.0.0-alpha1"
```
or something that allows us to release multiple alpha version.

Reply:
Going with `"6.0.0-alpha.1"` to follow guidelines here: https://go.dev/doc/modules/version-numbers 👍